### ISSUE TITLE
Installer for user fails: Permission denied

### ISSUE THREAD

**Issue by popstas** [viasite-ansible/ansible-role-zsh#19]
While provision role for root ansible creates directory in user directory with owner root.
Then provision from user fails:
```
TASK [viasite-ansible.zsh : Clone antigen master] ******************************
fatal: [localhost]: FAILED! => {"changed": false, "cmd": "/usr/bin/git clone --origin origin https://github.com/zsh-users/antigen.git /home/popstas/.antigen/antigen", "failed": true, "msg": "fatal: could not create work tree dir '/home/popstas/.antigen/antigen': Permission denied", "rc": 128, "stderr": "fatal: could not create work tree dir '/home/popstas/.antigen/antigen': Permission denied\n", "stderr_lines": ["fatal: could not create work tree dir '/home/popstas/.antigen/antigen': Permission denied"], "stdout": "", "stdout_lines": []}
        to retry, use: --limit @/tmp/zsh.retry
```

**Comment by popstas** [comment]
Quick fix:
```
$ sudo chown -R popstas:popstas .ansible .ansible_galaxy .antigen .antigen_plugin_lastupdate .antigen_system_lastupdate .bookmarks .cache
```

**Comment by popstas** [comment]
Fixed in #29 ?
